src/js/_enqueues/lib/comment-reply.js

```diff
@@ -292,8 +293,9 @@ window.addComment = ( function( window ) {
 	 * @param {String} commentId  Database ID of comment being replied to.
 	 * @param {String} respondId  HTML ID of 'respond' element.
 	 * @param {String} postId     Database ID of the post.
+	 * @param {String} replyTo    Form heading content.
 	 */
-	function moveForm( addBelowId, commentId, respondId, postId ) {
+	function moveForm( addBelowId, commentId, respondId, postId, replyTo ) {
```

Comment:
As above, it would be good to set a default `replyTo` in here (remember, no es6 so it will need to be old school).